app/assets/stylesheets/application.css.scss

```diff
@@ -124,16 +132,33 @@ input {
   border-color: white;
   border: 1px;
   border-radius: 4px;
+  height: 75%;
+  margin-top: 2%;
+  margin-right: 2%;
+  margin: 0 auto;
 }
 
 #results:empty {
   display: none;
 }
 
-.image-container > img {
+#content > img {
   // max-height: 200px;
-  height: 100%;
-  width: 100%;
+  width: 80%;
```

Comment:
Properties should be ordered border, border-color, border-radius, border-style, margin, margin-top, width